app/controllers/api/deploys_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Api::DeploysController < Api::BaseController
+  include CurrentProject
+
+  def index
+    scope = current_project.try(:deploys) || Deploy
+    @deploys =
+      if params[:ids]
+        Kaminari.paginate_array(scope.find(params[:ids])).page(1).per(1000)
```

Comment:
should fetch all when you give it a set of ids ? ... bad_request when you give it > 100